pkgs/applications/misc/blender/default.nix

```
@@ -26,11 +26,11 @@ let
 in
 stdenv.mkDerivation rec {
   pname = "blender";
-  version = "2.93.0";
+  version = "2.93.1";
 
   src = fetchurl {
     url = "https://download.blender.org/source/${pname}-${version}.tar.xz";
-    sha256 = "0f2rpqa39sir6g90khd2d2fs4kss0zhk7vya1nscf5yp8r566fxs";
+    sha256 = "sha256-IdriOBw/DlpH6B0GKqC1nKnhTZwrIL8U9hkMS20BHNg=";
```

Not sure if this checksum is correct:

```suggestion
    sha256 = "1n0w05nln30ryqaby81bki6y3acwnnh2l1hxx13ml3iz3hwf5ni1";
```

---

```
# nix hash to-base32 sha256-IdriOBw/DlpH6B0GKqC1nKnhTZwrIL8U9hkMS20BHNg=
1n0w05nln30ryqaby81bki6y3acwnnh2l1hxx13ml3iz3hwf5ni1
```